bundles/net.tourbook/src/net/tourbook/ui/tourChart/ChartLayerPause.java

```diff
@@ -43,10 +43,9 @@
 
    private ChartLabelPause  _hoveredLabel;
    private ChartLabelPause  _tooltipLabel;
-
    private int              _devXPause;
-   private int              _devYPause;
 
+   private int              _devYPause;
    private TourChart        _tourChart;
```

Comment:
This reformatting in your code was propably not intended

![image](https://user-images.githubusercontent.com/1283445/133877996-7c52c939-fb9a-4115-80e7-8eedf8d6b1be.png)

Reply:
No I did not. Maybe Eclipse did something without asking me?. Fixed in my [PR](https://github.com/wolfgang-ch/mytourbook/pull/398)